core/contacts.go

```diff
@@ -20,14 +20,16 @@ type ContactInfo struct {
 // AddContact adds a contact for the first time
 // Note: Existing contacts will not be overwritten
 func (t *Textile) AddContact(id string, address string, username string) error {
-	if err := t.datastore.Contacts().Add(&repo.Contact{
+	err := t.datastore.Contacts().Add(&repo.Contact{
 		Id:       id,
 		Address:  address,
 		Username: username,
 		Added:    time.Now(),
-	}); err != nil {
+	})
+	if err != nil {
```

Comment:
no need to log it here if we're returning it, could just return the datastore call... `return t.data....`

Reply:
yup